pydicom/config.py

```diff
@@ -86,9 +86,12 @@ def DS_decimal(use_Decimal_boolean=True):
 Default ``False``
 """
 
-empty_value = None
-"""Defines the value that an empty data element is set to
-after decoding."""
+use_none_as_empty_value = False
+""" If ``True``, the value of decoded empty data element is always ``None``.
+If ``False`` (the default), the value of an empty data element with 
+a text VR is an empty string, for all other VRs it is also ``None``.
+Note that the default of this value will change to ``True`` in version 1.5.
+"""
```

Comment:
Perhaps version 2.0 based on discussion in [Python 2 support thread](https://github.com/pydicom/pydicom/issues/872#issuecomment-516194302)?

Reply:
Yes, I wasn't sure about this anyway. 2.0 sounds good.